VirtoCommerce.Platform.Data/Repositories/Migrations/201908221604169_DynamicPropertyNameAddDescription.cs

```diff
@@ -0,0 +1,40 @@
+namespace VirtoCommerce.Platform.Data.Repositories.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class DynamicPropertyNameAddDescription : DbMigration
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.PlatformDynamicPropertyName", "Description", c => c.String(maxLength: 256));
+
+            Sql(@"DECLARE @cursor CURSOR
```

Comment:
I think a better solution will be using simple SQL Update statement instead of use Cursor and Fetch here

Reply:
Replaced.